cmd/kubeadm/app/features/features.go

```diff
@@ -41,6 +41,9 @@ const (
 
 	// SupportIPVSProxyMode is alpha in v1.8
 	SupportIPVSProxyMode = "SupportIPVSProxyMode"
+
+	// DynamicKubeletConfig is alpha in v1.9
+	DynamicKubeletConfig = "DynamicKubeletConfig"
```

Comment:
Why not `import "k8s.io/kubernetes/pkg/features"`, and avoid duplicating these feature gate key definitions?